[2024-09-02T16:17:00.314+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T16:17:00.325+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:16:59.464377+00:00 [queued]>
[2024-09-02T16:17:00.331+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:16:59.464377+00:00 [queued]>
[2024-09-02T16:17:00.332+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T16:17:00.340+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): first_task> on 2024-09-02 16:16:59.464377+00:00
[2024-09-02T16:17:00.347+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4008) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T16:17:00.349+0000] {standard_task_runner.py:72} INFO - Started process 4009 to run task
[2024-09-02T16:17:00.347+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'manual__2024-09-02T16:16:59.464377+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp748zsjin']
[2024-09-02T16:17:00.350+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask first_task
[2024-09-02T16:17:00.396+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:16:59.464377+00:00 [running]> on host cedbab6b7256
[2024-09-02T16:17:00.469+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T16:16:59.464377+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T16:16:59.464377+00:00'
[2024-09-02T16:17:00.471+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T16:17:00.482+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T16:17:00.483+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-02T16:17:00.490+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T16:17:00.493+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-02T16:17:00.494+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T16:17:00.510+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T16:17:00.511+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=manual__2024-09-02T16:16:59.464377+00:00, execution_date=20240902T161659, start_date=20240902T161700, end_date=20240902T161700
[2024-09-02T16:17:00.525+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T16:17:00.539+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-02T16:17:00.541+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
